app/src/lib/rules/case_frame/verbs.js

```diff
@@ -20,6 +20,7 @@ const default_verb_case_frame_json = {
 		'by_adposition': 'to',
 	},
 	'instrument': {
+		'trigger': 'none',
```

Comment:
is `none` different than not present?

Reply:
Yes, because of [this line](https://github.com/presciencelabs/tabitha-editor/blob/34ba7be8f89b7a289d26529ffaeabbd34b892e5a/app/src/lib/rules/case_frame/common.js#L81) in `case_frame/common.js`. That line allows a word like [be-T](https://github.com/presciencelabs/tabitha-editor/blob/34ba7be8f89b7a289d26529ffaeabbd34b892e5a/app/src/lib/rules/case_frame/verbs.js#L277) to 'clear' a default argument filter so it doesn't overlap with another one. But doing so broke the check for the 'instrument' (which I didn't notice until now). Using `'trigger': 'none'` fixed that.

Reply:
it's not important for this PR, maybe we can meet and see if there's a way to make this a little more self-documenting.